Formula/texlive.rb

```diff
@@ -30,20 +23,34 @@ class Texlive < Formula
   depends_on "perl"
 
   def install
-    scheme = %w[full medium small basic minimal].find do |x|
-      build.with? x
-    end || "small"
-
     ohai "Downloading and installing TeX Live. This will take a few minutes."
     ENV["TEXLIVE_INSTALL_PREFIX"] = libexec
-    system "./install-tl", "-scheme", scheme, "-portable", "-profile", "/dev/null"
+    system "./install-tl", "-scheme", "small", "-portable", "-profile", "/dev/null"
 
     man1.install Dir[libexec/"texmf-dist/doc/man/man1/*"]
     man5.install Dir[libexec/"texmf-dist/doc/man/man5/*"]
     rm Dir[libexec/"bin/*/man"]
     bin.install_symlink Dir[libexec/"bin/*/*"]
   end
 
+  def caveats; <<~EOS
+    To remove default scheme-small you may run:
+
+      tlmgr remove scheme-small
+
+    To install scheme-minimal you may run:
+
+      tlmgr install scheme-minimal
+
+    All possible schemes:
+
+      "scheme-full" to install everything
+      "scheme-medium" to install scheme-small + more packages and languages
+      "scheme-small" to install scheme-basic + xetex, metapost, a few languages [default]
+      "scheme-basic" to install plain and latex
+      "scheme-minimal" to install plain only
```

Comment:
I suggest replacing the above text with:
```suggestion
      The small (~500 MB) distribution of TexLive is installed by default.
      You may install a larger (medium or full) distribution of TexLive using one of:
        tlmgr install scheme-medium # 1.5 GB
        tlmgr install scheme-full # 6 GB
```

Reply:
Yes, you give rational sense? Minnimal and basic already present…